<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TilikausiSivu</class>
 <widget class="QWidget" name="TilikausiSivu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>576</width>
    <height>654</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Anna tiedot ensimmäisestä tilikaudesta, jonka kirjaat ja jonka tilinpäätöksen teet tällä ohjelmalla.&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Edellisen tilikauden &lt;/span&gt;tilinpäätöstietoja tarvitaan tilinpäätöksen lakisääteisiä vertailutietoja ja tilinavausta varten. Edellisen tilinpäätöksen tiedot syötetään ohjelmaan myöhemmin. Kirjanpidon voi aloittaa jo ennen edellisen tilikauden tietojen syöttämistä, jos esimerkiksi tilinpäätös ei ole vielä valmistunut.&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="font">
      <font>
       <italic>false</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="title">
      <string>Ensimmäinen Kitsaalla pidettävä tilikausi</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QFormLayout" name="formLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="font">
           <font>
            <italic>false</italic>
            <bold>false</bold>
           </font>
          </property>
          <property name="text">
           <string>Alkaa</string>
          </property>
          <property name="buddy">
           <cstring>ekaAlkaa</cstring>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QDateEdit" name="ekaAlkaa">
          <property name="font">
           <font>
            <italic>false</italic>
            <bold>false</bold>
           </font>
          </property>
          <property name="calendarPopup">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_2">
          <property name="font">
           <font>
            <italic>false</italic>
            <bold>false</bold>
            <kerning>false</kerning>
           </font>
          </property>
          <property name="text">
           <string>Päättyy</string>
          </property>
          <property name="buddy">
           <cstring>ekaPaattyy</cstring>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QDateEdit" name="ekaPaattyy">
          <property name="font">
           <font>
            <italic>false</italic>
            <bold>false</bold>
           </font>
          </property>
          <property name="calendarPopup">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QCheckBox" name="aloittavaTilikausiCheck">
        <property name="font">
         <font>
          <italic>false</italic>
          <bold>false</bold>
          <kerning>false</kerning>
         </font>
        </property>
        <property name="text">
         <string>Toiminta alkaa vasta tästä tilikaudesta (ei tilinavausta)</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="eiavaustaVaroitus">
     <property name="styleSheet">
      <string notr="true">background-color: yellow;
color: rgb(0, 0, 0);</string>
     </property>
     <property name="text">
      <string>&lt;p&gt;&lt;b&gt;Et voi syöttää ohjelmaan taseen alkusaldoja, koska olet valinnut että toiminta alkaa vasta tästä tilikaudesta.&lt;/b&gt;&lt;/p&gt;
&lt;p&gt;Käytä tätä valintaa vain, jos kyseessä on todellakin vasta perustetun yrityksen/yhdistyksen ensimmäinen tilikausi.&lt;/p&gt;</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="font">
      <font>
       <bold>true</bold>
      </font>
     </property>
     <property name="title">
      <string>Edellinen tilikausi</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QFormLayout" name="formLayout_2">
        <item row="0" column="0">
         <widget class="QLabel" name="label_3">
          <property name="font">
           <font>
            <bold>false</bold>
           </font>
          </property>
          <property name="text">
           <string>Alkoi</string>
          </property>
          <property name="buddy">
           <cstring>edellinenAlkoi</cstring>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QDateEdit" name="edellinenAlkoi">
          <property name="font">
           <font>
            <bold>false</bold>
           </font>
          </property>
          <property name="calendarPopup">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_4">
          <property name="font">
           <font>
            <bold>false</bold>
           </font>
          </property>
          <property name="text">
           <string>Päättyi</string>
          </property>
          <property name="buddy">
           <cstring>edellinenPaattyi</cstring>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QDateEdit" name="edellinenPaattyi">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="font">
           <font>
            <bold>false</bold>
           </font>
          </property>
          <property name="calendarPopup">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>42</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>aloittavaTilikausiCheck</sender>
   <signal>toggled(bool)</signal>
   <receiver>eiavaustaVaroitus</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>66</x>
     <y>318</y>
    </hint>
    <hint type="destinationlabel">
     <x>63</x>
     <y>381</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>aloittavaTilikausiCheck</sender>
   <signal>toggled(bool)</signal>
   <receiver>groupBox_2</receiver>
   <slot>setHidden(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>391</x>
     <y>317</y>
    </hint>
    <hint type="destinationlabel">
     <x>402</x>
     <y>575</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
